src/main/java/uk/gov/pay/adminusers/service/ServiceUpdater.java

```diff
@@ -79,7 +79,11 @@ public Service doUpdateMerchantDetails(String serviceExternalId, UpdateMerchantD
     }
 
     private BiConsumer<ServiceUpdateRequest, ServiceEntity> updateServiceName() {
-        return (serviceUpdateRequest, serviceEntity) -> serviceEntity.setName(serviceUpdateRequest.valueAsString());
+        return (serviceUpdateRequest, serviceEntity) -> {
+            serviceEntity.setName(serviceUpdateRequest.valueAsString());
+            serviceEntity.getServiceNames()
+                    .put(SupportedLanguage.ENGLISH, ServiceNameEntity.from(SupportedLanguage.ENGLISH, serviceUpdateRequest.valueAsString()));
```

Comment:
I think for the ORM magic to work, this should update the existing `ServiceNameEntity` if there already is one rather than creating a new one every time, in a similar fashion to `ServiceNameEntity.addOrUpdateServiceName(ServiceNameEntity)`. In fact, we can probably just call that method (which will also update the `name` because of the logic already in there to update the `name` when the `service_name/en` is updated).